apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_state/states/SellerState3a.java

```diff
@@ -154,12 +157,27 @@ private void onBtcSent() {
             String name = model.getBisqEasyTrade().getContract().getBaseSidePaymentMethodSpec().getPaymentMethod().getPaymentRail().name();
             String proof = Res.get("bisqEasy.tradeState.info.seller.phase3a.tradeLogMessage.paymentProof." + name);
             String userName = model.getChannel().getMyUserIdentity().getUserName();
-            if (model.getPaymentProof().get() == null) {
+            String txId = model.getPaymentProof().get();
+            if (txId == null) {
                 sendTradeLogMessage(Res.encode("bisqEasy.tradeState.info.seller.phase3a.tradeLogMessage.noProofProvided", userName));
```

Comment:
sendTradeLogMessage should be moved into sellerConfirmedBtcSent.

Reply:
to be addressed in the next amend